eagleproject/core/blockchain/harvest/transaction_history.py

```diff
@@ -641,18 +795,22 @@ def fetch_ogv_data(to_block, from_timestamp, to_timestamp):
         'percentage_staked': percentage_staked,
     }
 
-def get_curve_data(to_block):
-    balance = balanceOf(CURVE_METAPOOL, CURVE_METAPOOL_GAUGE, 18, to_block)
-    supply = totalSupply(CURVE_METAPOOL, 18, to_block)
-    return {
-        "total_supply": supply,
-        "earning_ogn": balance/supply,
-    }
-
-def create_time_interval_report(from_block, to_block, from_block_time, to_block_time, project=OriginTokens.OUSD):
-    if project == OriginTokens.OETH:
-        raise Exception("Unimplemented for OETH")
+def get_curve_data(to_block, project=OriginTokens.OUSD):
+    if project == OriginTokens.OUSD:
+        balance = balanceOf(CURVE_METAPOOL, CURVE_METAPOOL_GAUGE, 18, to_block)
+        supply = totalSupply(CURVE_METAPOOL, 18, to_block)
+        return {
+            "total_supply": supply,
+            "earning_ogn": balance/supply,
+        }
+    else:
+        # balance = balanceOf(OETH_ETH_AMO_METAPOOL, OETH_ETH_AMO_CURVE_GUAGE, 18, to_block)
```

Comment:
Nope, this isn't used anywhere. Even for OUSD, OGN buyback no longer happens so that seems irrelevant. Should probably remove that as well but I don't wanna risk breaking anything with this PR